sympy/stats/frv_types.py

```diff
@@ -102,6 +109,10 @@ def pmf(self, x):
         else:
             return S.Zero
 
+    def _sample_random(self, size):
+        return [self.args[random.randint(0, len(self.args)-1)] for _ in range(size)]
```

Comment:
this one should use `ArrayComprehensionMap`

Reply:
@Upabjojr I have made the changes, please review.